CMakeLists.txt

```diff
@@ -18,8 +18,9 @@
 cmake_minimum_required(VERSION 2.8)
 
 include(cmake/base.cmake)
-include(cmake/eigen.cmake)
 include(cmake/boost.cmake)
+include(cmake/cpack.cmake)
```

Comment:
I'm pretty sure this is not needed. See comments in `cpack.cmake` for better altenratives.

Reply:
This line has been removed.